api/src/main/java/com/codeforcommunity/api/IProtectedSiteProcessor.java

```diff
@@ -23,13 +25,22 @@ public interface IProtectedSiteProcessor {
   /** Removes the record in the adopted sites table linking the site to its current adopter */
   void forceUnadoptSite(JWTData userData, int siteId);
 
+  /** Adopts a tree for a child account through the parent account **/
```

Comment:
Are these purpose statements doing more than what the function name already provides?